ansible_collections/azureredhatopenshift/cluster/roles/smoketest/tasks/new_project.yaml

```diff
@@ -0,0 +1,210 @@
+#  oc project default (this test assumes we are using the default ns)
+#  oc new-app --image=nginx && oc expose svc/nginx
+#  oc get deployment,pod,route,svc
+#  curl -I --silent http://<hostname you get from the route>
+
+# "Apply the following yml file:
+# apiVersion: v1
+# kind: PersistentVolumeClaim
+# metadata:
+#   name: azurefile-csi
+# spec:
+#   storageClassName: azurefile-csi
+#   accessModes:
+#     - ReadWriteOnce
+#   resources:
+#     requests:
+#       storage: 3Gi
+# ---
+# apiVersion: v1
+# kind: PersistentVolumeClaim
+# metadata:
+#   name: managed-csi
+# spec:
+#   storageClassName: managed-csi
+#   accessModes:
+#     - ReadWriteOnce
+#   resources:
+#     requests:
+#       storage: 3Gi
+# ---
+# apiVersion: v1
+# kind: PersistentVolumeClaim
+# metadata:
+#   name: managed-csi-encrypted-cmk
+# spec:
+#   storageClassName: managed-csi-encrypted-cmk
+#   accessModes:
+#     - ReadWriteOnce
+#   resources:
+#     requests:
+#       storage: 3Gi
+# "
```

Comment:
Should this be removed?

Reply:
This, and several similar comments were describing the manual steps from the smoketest spreadsheet that was getting automated.